src/api/nodeRpcCall.js

```diff
@@ -1578,36 +1608,64 @@ const senateProposeSpend = async ({
   return submitExtrinsic(extrinsic, walletAddress, api);
 };
 
+const getAdminMinistryFinance = async () => {
+  const api = await getApi();
+  return api.query.ministryOfFinanceOffice.admin();
```

Comment:
admin appoints clerks

Clerks actually send the funds.

I assume you need clerks, not admin?